### DISCUSSION TITLE
How do I install this?

### DISCUSSION THREAD

**Discussion by JustAskinByMe** [CyferShepard/Jellystat#176] [Q&A]
Run npm install to install necessary packages
Run npm run start-server to only run the backend nodejs server
Run npm run start to only run the frontend React UI
Run npm run start-app to run both backend and frontend at the same time

I see that here, but I've downloaded the files and cannot find npm install.
If it makes a difference I installed jellyfin with the windows exe file

**Comment by CyferShepard** [comment]
Hi @JustAskinByMe , these instructions are to setup a development environment, not to install Jellystat. Currently its only supported on docker, which can be installed on most platforms. You can find the docker instructions here https://hub.docker.com/r/cyfershepard/jellystat
